### DISCUSSION TITLE
How to combine `Stdin` and `Stdout` into a bidirectional stream?

### DISCUSSION THREAD

**Discussion by ncihnegn** [tokio-rs/tokio#6215] [Q&A] [answered]
So I can use `copy_bidirectional` to redirect to a `TcpStream`.

**Comment by ncihnegn** [comment]
I found `readwrite::ReadWriteTokio`. Is there anything similar in `tokio`?

**Reply by Darksonn** [reply, marked as answer]
I don't think we provide a utility for this. But it would be a reasonable addition to tokio-util.
